esmanager.py

```diff
@@ -12,6 +12,10 @@
 INDEX = "locations"
 TYPE = "locations"
 
+# If the database size would decrease by this number or more,
+# require the --force flag
+SANITY_THRESHOLD = 100
```

Comment:
What do you think a good threshold would be? I picked 100 arbitrarily, but I didn't put much thought into it.

Reply:
@magical I would say let's set it much lower.. let's start with `5`. Given the types of locations (parking, buildings, restaurants), and the frequency that this script runs (every 5 minutes), it would be very suspicious if more than 5 locations were _supposed_ to be deleted in a given run. If it starts to become an issue, we can always increase it. I'd just rather start low.